src/Component/HomePage.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react'
```

Comment:
This whole component was not required. You should have rendered a PDF of your resume instead.